starter-code/css/main.css

```diff
@@ -6,3 +6,152 @@ Green: #00B172
 White: #FFF
 
 */
```

Comment:
As you may have notice, there is margin on the top of your document (even if you didn't add it), it's because many browser already have a default style, to avoid this you can rewrite this default value by adding this few lines on the top of your document : 

```css
* {
  margin: 0 auto;
  padding: 0 auto;
  font-family: MOntserrat;
}
```